Userland/Utilities/top.cpp

```diff
@@ -203,11 +203,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(Core::System::unveil("/etc/passwd", "r"));
     unveil(nullptr, nullptr);
 
-    signal(SIGWINCH, [](int) {
+    TRY(Core::System::signal(SIGWINCH, [](int) {
         g_window_size_changed = true;
-    });
+    }));
+    TRY(Core::System::signal(SIGINT, [](int) {
+        exit(0);
+    }));
 
-    TRY(Core::System::pledge("stdio rpath tty"));
```

Comment:
Why is this removed?

Reply:
It was overwriting the past pledge at the top of the function.

Reply:
Sorry, I got an error when testing about a missing pledge. This line has been re-added.